examples/Dashboard/src/ReplenishmentWidget.tsx

```diff
@@ -65,12 +62,12 @@ const ReplenishmentWidget: React.FC<PropsWithChildrenOnly> = () => {
           flex={1}
           flexDirection="column"
         >
-          <Grid item>
+          <Grid>
             <StatsPanel
               error={error as ApiException}
               isError={isError}
               isLoading={isLoading}
-              title={t('inbound-shipments', { ns: 'app' })}
+              title={t('inbound-shipment', { ns: 'app' })}
```

Comment:
This was actually changed, and also namespace should be removed: https://github.com/msupply-foundation/open-msupply-plugins/issues/6